packages/yew-macro/src/function_component.rs

```diff
@@ -144,10 +147,96 @@ impl Parse for FunctionComponent {
             name: sig.ident,
             return_type,
             fn_token: sig.fn_token,
+            component_name: None,
         })
     }
 }
 
+impl FunctionComponent {
+    /// Filters attributes that should be copied to component definition.
+    fn filter_attrs_for_component_struct(&self) -> Vec<Attribute> {
+        self.attrs
+            .iter()
+            .filter_map(|m| {
+                m.path
+                    .get_ident()
+                    .and_then(|ident| match ident.to_string().as_str() {
+                        "doc" | "allow" => Some(m.clone()),
```

Comment:
We should pass `deny` and `warn`. if we're passing along `allow`, or does it not matter?

Reply:
We probably shouldn't. This makes users to be able to apply lints on generated code.

Lints are usually suppressed on generated code.